docs/README.md

```diff
@@ -1,272 +1,278 @@
-- [Google Sign-In](./people-and-groups/google-sign-in.md)
-- [LDAP](./people-and-groups/ldap.md)
-- [API keys](./people-and-groups/api-keys.md)
+* [People overview](./people-and-groups/start.md)
+* [Account settings](./people-and-groups/account-settings.md)
+* [Managing people and groups](./people-and-groups/managing.md)
+* [Password complexity](./people-and-groups/changing-password-complexity.md)
+* [Session expiration](./people-and-groups/changing-session-expiration.md)
+* [Google Sign-In](./people-and-groups/google-sign-in.md)
+* [LDAP](./people-and-groups/ldap.md)
+* [API keys](./people-and-groups/api-keys.md)
 
 #### Paid SSO options
 
-- [JWT-based authentication](./people-and-groups/authenticating-with-jwt.md)
-- [SAML-based authentication](./people-and-groups/authenticating-with-saml.md)
-  - [SAML with Auth0](./people-and-groups/saml-auth0.md)
-  - [SAML with Microsoft Entra ID](./people-and-groups/saml-azure.md)
-  - [SAML with Google](./people-and-groups/saml-google.md)
-  - [SAML with Keycloak](./people-and-groups/saml-keycloak.md)
-  - [SAML with Okta](./people-and-groups/saml-okta.md)
-- [User provisioning with SCIM](./people-and-groups/user-provisioning.md)
+* [JWT-based authentication](./people-and-groups/authenticating-with-jwt.md)
+* [SAML-based authentication](./people-and-groups/authenticating-with-saml.md)
+
+  * [SAML with Auth0](./people-and-groups/saml-auth0.md)
+  * [SAML with Microsoft Entra ID](./people-and-groups/saml-azure.md)
+  * [SAML with Google](./people-and-groups/saml-google.md)
+  * [SAML with Keycloak](./people-and-groups/saml-keycloak.md)
+  * [SAML with Okta](./people-and-groups/saml-okta.md)
+* [User provisioning with SCIM](./people-and-groups/user-provisioning.md)
 
 ### Permissions
 
-- [Permissions overview](./permissions/start.md)
-- [Permissions introduction](./permissions/introduction.md)
-- [Data permissions](./permissions/data.md)
-- [Collection permissions](./permissions/collections.md)
-- [Application permissions](./permissions/application.md)
-- [Row and column security](./permissions/row-and-column-security.md)
-- [Row and column security examples](./permissions/row-and-column-security-examples.md)
-- [Connection impersonation](./permissions/impersonation.md)
-- [Database routing](./permissions/database-routing.md)
-- [Snippets folder permissions](./permissions/snippets.md)
-- [Notification permissions](./permissions/notifications.md)
-- [Configuring permissions for embedding](./permissions/embedding.md)
+* [Permissions overview](./permissions/start.md)
+* [Permissions introduction](./permissions/introduction.md)
+* [Data permissions](./permissions/data.md)
+* [Collection permissions](./permissions/collections.md)
+* [Application permissions](./permissions/application.md)
+* [Row- and column-level security](./permissions/row-and-column-security.md)
+* [Row- and column-level security examples](./permissions/row-and-column-security-examples.md)
```

Comment:
"Row and column security" is the name of the feature in the UI. The link to docs about the feature should reflect the feature name.